content/articles/announcement-secondary-dns-ip-address-changes.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Secondary DNS IP Address Changes
+excerpt: DNSimple will be updating the IP addresses used for Secondary DNS outbound notifications and zone transfers. The current IPv4 address (50.31.225.92) will be retired beginning [DATE], 2024. Affected customers will be notified via email.
+categories:
+---
+
+# Secondary DNS IP Address Changes
+
+<info>
+This article contains information that is only relevant to customers with secondary DNS enabled, where DNSimple acts as the primary used for outbound notifications and zone transfers. If you are impacted, you will receive an email from us asking you to take action.
+</info>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+As part of our commitment to providing streamlined and reliable DNS management services, we are modernizing our secondary DNS infrastructure and updating the primary server IP addresses used for outbound notifications and zone transfers. **The current IPv4 address (50.31.225.92) will be retired beginning [DATE], 2024.**
```

Comment:
Remember to set the DATE.

Reply:
Set in https://github.com/dnsimple/dnsimple-support/pull/1258/commits/9f0f9f294517887d36cfa99483ccf0c87282f998